bridge/launchpad/import.go

```diff
@@ -70,6 +73,47 @@ func (li *launchpadImporter) ImportAll(repo *cache.RepoCache) error {
 			fmt.Println("TODO: Update bug")
 		}
 
+		/* Handle messages */
+		if len(lpBug.Messages) == 0 {
+			return errors.Wrapf(err, "failed to fetch comments for bug #%s", lpBugID)
+		}
+
+		// The Launchpad API returns the bug description as the first
+		// comment, so skip it.
+		for i, lpMessage := range lpBug.Messages[1:] {
+			commentID := fmt.Sprintf("%d-%d", lpBug.ID, i)
+			_, err := b.ResolveTargetWithMetadata(keyLaunchpadID, commentID)
+			if err != nil && err != cache.ErrNoMatchingOp {
+				return errors.Wrapf(err, "failed to fetch comments for bug #%s", lpBugID)
+			}
+
+			// If this comment already exists, we are probably
+			// updating an existing bug. We do not want to duplicate
+			// the comments, so let us just skip this one.
+			// TODO: Can Launchpad comments be edited?
+			if err == nil {
+				continue
+			}
+
+			// This is a new comment, we can add it.
+			createdAt, _ := time.Parse(time.RFC3339, lpMessage.CreatedAt)
+			err = b.AddCommentRaw(
+				li.makePerson(lpMessage.Owner),
+				createdAt.Unix(),
+				lpMessage.Content,
+				nil,
+				map[string]string{
+					keyLaunchpadID: commentID,
+				})
+			if err != nil {
+				return errors.Wrapf(err, "failed to add comment to bug #%s", lpBugID)
+			}
+			err = b.CommitAsNeeded()
```

Comment:
As discussed, the `CommitAsNeeded()` should probably be on the bug loop, not the comment loop, to reduce the overhead of commit, blob and all.